dns-metrics.schema.json

```diff
@@ -104,65 +100,65 @@
             "type": "object",
             "properties": {
                 "since": {
-                    "description": "",
+                    "description": "The starting time for when the metrics of this object was gathered, in nanoseconds since 1970-01-01 00:00:00 UTC",
                     "type": "integer"
                 },
                 "until": {
-                    "description": "",
+                    "description": "The ending time for when the metrics of this object was gathered, in nanoseconds since 1970-01-01 00:00:00 UTC",
                     "type": "integer"
                 },
-                "requests": {
-                    "description": "",
+                "queries": {
```

Comment:
It's not just queries, it can be also UPDATE and other kinds of requests.

Reply:
Ah true, but `requests` doesn't say anything about that and `queries`/`responses` can be tied directly to the QR bit.

Maybe add `qtypes` like `response_rcodes` ?